### PR TITLE
Us29b covnert json to xml

### PR DESCRIPTION
**Pull request by Kirosoft**
**\* do not merge ***

1) Addition of /xmlify which includes a third party library to build XML from python classes (modified to work with Python3)
2) Added a class into utils to generate XML from the XSD generated Python classes
3) Added another method in utils to validate the generated XML against the original XSD
4) Added a unit test to check the process

### CODE REVIEW

File: application/deed/schemas/Signature.xsd [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+  <sequence> 
+    <element ref="ds:SignedInfo"/> 
+    <element ref="ds:SignatureValue"/> 
+    <element ref="ds:KeyInfo" minOccurs="0"/> 
+    <element ref="ds:Object" minOccurs="0" maxOccurs="unbounded"/> 
+  </sequence>  
+  <attribute name="Id" type="ID" use="optional"/>
+</complexType>
+
+  <element name="SignatureValue" type="ds:SignatureValueType"/>
```
Comment (endamccormack) [inline application/deed/schemas/Signature.xsd]: de-Indent

**Comment by endamccormack** [comment]
Is this an autogenerated file? There is some strange indentation in it. https://github.com/LandRegistry/dm-deed-api/blob/us29b-covnert-json-to-xml/application/deed/schemas/Signature.xsd

**Comment by endamccormack** [comment]
this too https://github.com/LandRegistry/dm-deed-api/blob/us29b-covnert-json-to-xml/application/deed/schemas/deed-schema-v0-1.xsd

File: application/deed/utils.py [hunk 2]
```diff
@@ -101,4 +108,81 @@ def load_json_file(file_path):
     return json_data
 
 
+def convert_json_to_sig_slot(borrower_json):
```
Comment (endamccormack) [inline application/deed/utils.py]: I have no idea what a sig slot is, either better descriptive name or a comment please

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**